apps/jobs/utils/validators.py

```diff
@@ -167,21 +167,14 @@ def validate_email(email):
                             data[field_name] = "15+"
                     else:
                         raise ValidationError(f"Invalid {field_name} value provided")
+                elif field_name == "name" or field_name == "full_name":
```

Comment:
both (for name, fullname and message) these `elif` statements can be combined because they are doing the same thing. 
for example: 
```py
elif field_name in ["name", "full_name", "message"]:
    if condition:
        raise ValidationError({"error": f"Invalid {field_value} format specified"})
```